dbt/adapters/databricks/connections.py

```diff
@@ -98,9 +98,7 @@ def unique_field(self) -> str:
         return self.host
 
     def _connection_keys(self) -> Tuple[str, ...]:
-        connection_keys = ["host", "http_path", "schema"]
-        if self.database:
```

Comment:
The connection_keys seems to be used here: https://github.com/dbt-labs/dbt-core/blob/8f50eee330ea34dd6aaba40993a01a380373fa32/core/dbt/contracts/connection.py#L144-L153

@jtcohen6 do you know if we can use `catalog` instead of `database` as a connection key here? 
```
Connection:
  host: your.databrickshost.com
  http_path: /sql/your/http/path
  database: catalog_name  <----- change this to "catalog: catalog_name"
  schema: schema_name
```

Reply:
I submitted another PR with a hacky way (#145).

Reply:
Ah - I don't think there's a good way to do this, based on the way we've implemented aliases (always resolves to the "true" name within dbt). I think the hacky way is good enough for now